privacyidea/lib/crypto.py

```diff
@@ -73,15 +74,17 @@
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives.asymmetric import padding as asym_padding
 
-import passlib.hash
-from passlib.hash import argon2
-
 
 if not PY2:
     long = int
 
+# The CryptContext makes it easier to work with multiple password hash algorithms:
+# The first algorithm in the list is the default algorithm used for hashing.
+# When verifying a password hash, all algorithms in the context are checked
+# until one succeeds (or all fail).
+pass_ctx = CryptContext(['argon2', 'pbkdf2_sha512'], argon2__rounds=9)
```

Comment:
A bit nitpicking: Could you put the 

    ROUNDS=9

at the top of the file and the add ``argon2__rounds=ROUNDS``.
This might be ease the change of the number of rounds later.

Reply:
Done in 5b0334d7